### PR TITLE
`SECURITY.md`

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `SECURITY.md` added
- Minor edit in `AUTHORS.md`
- Minor edit in `README.md`

### CODE REVIEW

File: AUTHORS.md
Comment (AHReccese) [inline AUTHORS.md:1]: Why did we have a `#` at the end of the titles previously?

File: otherfiles/RELEASE.md
Comment (AHReccese) [inline otherfiles/RELEASE.md:1]: Same question here, I'm just wondering the reason behind although I'm totally fine with the new one.

**Review by AHReccese — COMMENTED** [review]
I approve the changes, but I have some tiny questions, so let me know the answers!

File: AUTHORS.md
Comment (sepandhaghighi) [reply AUTHORS.md:1]: Some `markdown` editors, such as `MarkdownPad`, use a non-standard style of heading like this.

**Comment by AHReccese** [comment]
Thank you for your answer.
Interesting, so I think we should update all of our other projects too.

**Review by AHReccese — APPROVED** [review]

**Pull request merged**